frontend/src/store/eam-group-configurate/reducer.ts

```diff
@@ -20,6 +25,10 @@ const reducer = createReducer(initialState, (builder) => {
     state.dataStatus = DataStatus.FULFILLED;
     state.workers = action.payload.items;
   });
+  builder.addCase(getPermission.fulfilled, (state, action) => {
+    state.dataStatus = DataStatus.FULFILLED;
```

Comment:
we should have data status for each entity for loading (ex: PermissionsDataStatus)